shell/common/animator.cc

```diff
@@ -185,7 +185,10 @@ void Animator::Render(std::unique_ptr<flutter::LayerTree> layer_tree) {
   }
 
   // Commit the pending continuation.
-  producer_continuation_.Complete(std::move(layer_tree));
+  bool result = producer_continuation_.Complete(std::move(layer_tree));
+  if (!result) {
+    FML_DLOG(INFO) << "No pending continuation to commit";
```

Comment:
I was considering `FML_DCHECK` the result, but some shell_tests fails the check.
I suppose it is expected that `Complete` fails here? Or do we always expect the `Complete` to success here? (meaning the `continuation_` is never null here.) 
@chinmaygarde

Reply:
I am not able to reason about when that would fail and if on failure, the delegates OnAnimatorDraw invocation needs to be made. cc @iskakaushik 

I suppose it is fine to log for now as this was not the point of your change.